example/src/Utilities/APIClient.js

```diff
@@ -36,11 +36,12 @@ class ApiClient {
     return ApiClient.makeRequest(ENVIRONMENT.url + 'sessions', body);
   };
 
-  static paymentMethods = configuration => {
+  static paymentMethods = (configuration, order) => {
     const body = {
       ...parseConfig(configuration),
       ...parseAmount(configuration),
       ...serverConfiguration,
+      order: order ? parseOrder(order) : undefined,
```

Comment:
I think we can omit the order key if there is no `order`. So that we don't include `order: undefined` in the body.

```js
...(order && { order: parseOrder(order) }),
```

Reply:
Fixed on [4513c5b](https://github.com/Adyen/adyen-react-native/pull/542/commits/4513c5b173e7e9d053e941bfc33777b423d7404a)